practice.css

```diff
@@ -158,7 +158,7 @@ h2.subtitle {
 
 .nameModal{
   background-color: antiquewhite;
-  height: 40%;
+  height: 22.5%;
```

Comment:
You have to remove this property.
DONOT change the value as this creates unresponsiveness.
SEE: 
![image](https://user-images.githubusercontent.com/41796013/198868301-98e805c6-4fec-4665-81f0-1c95e8228840.png)

Reply:
ook trying to do that